### PR TITLE
allow to define service type in Kafka/ZK

### PR DESCRIPTION
**Pull request by stas2k**
These changes allow to define type of exposed Kafka/ZooKeeper services.

### CODE REVIEW

**Comment by ghost** [comment]
related: https://github.com/kubernetes/charts/pull/3754

**Comment by stas2k** [comment]
@josdotso thanks, I like your Kafka solution

File: incubator/kafka/templates/service-brokers.yaml [hunk 1]
```diff
@@ -14,3 +14,10 @@ spec:
   selector:
     app: {{ include "kafka.name" . | quote }}
     release: {{ .Release.Name | quote }}
+  type: {{.Values.serviceType}}
+  {{if eq .Values.serviceType "LoadBalancer"}}
```
Comment (unguiculus) [inline incubator/kafka/templates/service-brokers.yaml:18]: For consistent formatting, please add a space after `{{` and before `}}`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
related: https://github.com/kubernetes/charts/pull/4931

**Comment by unguiculus** [comment]
Please consider splitting this up into two PRs. Also, #4931 will likely make it in.

**Comment by unguiculus** [comment]
@stas2k Are going to update the PR? Otherwise, please close it. Thanks.

**Pull request closed without merging**